Source/WebCore/platform/text/LocaleICU.cpp

```diff
@@ -188,11 +188,7 @@ std::unique_ptr<Vector<String>> LocaleICU::createLabelVector(const UDateFormat*
 
 static std::unique_ptr<Vector<String>> createFallbackMonthLabels()
 {
-    auto labels = makeUnique<Vector<String>>();
-    labels->reserveInitialCapacity(std::size(WTF::monthFullName));
-    for (auto monthName : WTF::monthFullName)
-        labels->uncheckedAppend(monthName);
-    return labels;
+    return makeUnique<Vector<String>>(WTF::monthFullName, 12);
```

Comment:
Why 12 instead of `std::size`? Could we do an overload that automatically does `std::size`?

Reply:
Some code was using 12, some was using std::size(). I decided on 12 because this is a number of month so the size of these arrays is not going to change. I can look into the overload, or using std::size() more consistently.

Reply:
If some already used 12, I am OK not pushing to use `std::size` everywhere.